google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/ManagedCloudSdkServiceUiPresenter.java

```diff
@@ -51,9 +52,16 @@ static void setInstance(ManagedCloudSdkServiceUiPresenter uiPresenter) {
     ManagedCloudSdkServiceUiPresenter.instance = uiPresenter;
   }
 
-  public void notifyManagedSdkJobSuccess(ManagedSdkJobType jobType) {
+  public void notifyManagedSdkJobSuccess(ManagedSdkJobType jobType, ManagedSdkJobResult jobResult) {
     String message = GctBundle.message("managedsdk.success." + jobType.name().toLowerCase());
-    showNotification(message, NotificationType.INFORMATION);
+    switch (jobResult) {
+      case PROCESSED:
+        showNotification(message, NotificationType.INFORMATION);
+        break;
+      case UP_TO_DATE:
```

Comment:
assuming we keep this enum: from what I remember from reading the style guide the other day, you should remove the UP_TO_DATE case since its handled by the default case (and you already have a nice comment there indicating that its up-to-date).

Also, the `break` above can be replaced by a `return` right? (which may be a bit more readable and resilient to changes)

Reply:
sounds good, that makes sense, updating.